README.md

```diff
@@ -26,3 +26,21 @@ Test coverage is woefully incomplete, but comes in three flavors:
 ## Installation
 
 See [INSTALL](INSTALL) for system requirements and instructions.
+
+# Development Docker
+
+A basic Docker setup is available to simplify setting up a development environment but requires a
+couple of manual steps to get the site up and running.
+
+1. `cp php/config.sample.php php/config.php` and set `$host = "db";` so the host name matches the
+   the database container host name.
+2. Run `docker-compose up` to create the containers.
+3. Install local PHP dependencies inside the container.
```

Comment:
Any reason we can't run composer automatically?

Reply:
We only mount the source code when the container is created. It might be possible as part of `CMD` but this wasn't a priority for me. If you have PHP installed locally, you can run it outside of Docker too.